provisioning/miniprov/test/test_cdf.py

```diff
@@ -119,6 +143,59 @@ def ret_values(value: str) -> Any:
         self.assertEqual(Text('eth1'), ret[0].data_iface)
         self.assertEqual(1, ret[0].s3_instances)
 
+        ret = CdfGenerator(provider=store)._create_pool_descriptions()
+        self.assertIsInstance(ret, list)
+        self.assertEqual(1, len(ret))
+        self.assertEqual(Text('StorageSet-1'), ret[0].name)
+        self.assertEqual(1, ret[0].data_units)
+        self.assertEqual(0, ret[0].parity_units)
+        disk_refs = ret[0].disk_refs.value
+        self.assertEqual(Text('srvnode_1'), disk_refs.value[0].node.value)
+        self.assertEqual(Text('/dev/sdb'), disk_refs.value[0].path)
+
+        ret = CdfGenerator(provider=store)._create_profile_descriptions(ret)
+        self.assertIsInstance(ret, list)
+        self.assertEqual(1, len(ret))
+        self.assertEqual(Text('Profile_the_pool'), ret[0].name)
+        self.assertEqual(1, len(ret[0].pools.value))
+        self.assertEqual(Text('StorageSet-1'), ret[0].pools.value[0])
+
+    def test_invalid_storage_set_configuration(self):
```

Comment:
Suggestion: rename the method to something like `test_invalid_storage_set_configuration_rejected`.

Reason: test names should (ideally) specify the statement about software unit or give a specific description of how the software is supposed to behave.

So if you have statement like "My software analyzes the correctness of storage set configuration AND it raises an exception if is incorrect", then it would be nice to give the corresponding clue in the test name. Note that unit tests can a kind of document the software and how it is supposed to work.

Reply:
It would be nice to put a comment within test method describing why exactly the given configuration is incorrect, it is not that obvious to me. That would help to maintain your test in the future.

Reply:
Done